apps/console/src/extensions/components/subscription/api/subscription.ts

```diff
@@ -42,9 +43,12 @@ const useTenantTier = <Data = TenantTierRequestResponse,
         url: getTenantResourceEndpoints().tenantSubscriptionApi + "/tier" + getDomainQueryParam()
     };
 
-    const { data, error, isValidating, mutate } = useRequest<Data, Error>(requestConfig, {
-        shouldRetryOnError: false
-    });
+    const { data, error, isValidating, mutate } = useRequest<Data, Error>(
+        Config.getDeploymentConfig().extensions?.subscriptionApiPath ? requestConfig : null,
```

Comment:
Is it a good practise to directly call the `Config` object from the API layer ?
cc: @brionmario

Reply:
we can use hooks here since this is a hook.  Can we get `subscriptionApiPath` from Redux or any other text?

Reply:
@pavinduLakshan lets check this in a follow up PR.